src/FluentValidation/Resources/Languages/EnglishLanguage.cs

```diff
@@ -46,6 +46,7 @@ internal class EnglishLanguage : Language {
 			Translate<EmptyValidator>("'{PropertyName}' must be empty.");
 			Translate<NullValidator>("'{PropertyName}' must be empty.");
 			Translate<EnumValidator>("'{PropertyName}' has a range of values which does not include '{PropertyValue}'.");
+			Translate<StringEnumValidator>("'{PropertyName}' has names which do not include '{PropertyValue}'.");

```

Comment:
Let's remove this message and use the `EnumValidator` message instead, so re-translation isn't necessary.